applications_tests/gls_sharp_navier_stokes_3d/sphere_mms_pressure_scaling.output

```diff
@@ -0,0 +1,35 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       64
+   Number of degrees of freedom: 500
+   Volume of triangulation:      8
+
+*****************************
+Steady iteration:        1/3
+*****************************
+div u : 0
+error at the weak BC : 0
+L2 error velocity : 0 L2 error pressure: 0
```

Comment:
Why is the error here zero? I would remove this iteration or this mesh if it is inadequate.

Reply:
That's because every cell was either cut or overconstrained, so the error calculation took into account 0 cell. I'll remove this first level.